pybossa/cache/task_browse_helpers.py

```diff
@@ -187,8 +187,7 @@ def get_searchable_columns(project_id):
                   WHERE jsonb_typeof(info) = 'object'
                   AND project_id=:project_id;''')
     results = db.slave_session.execute(sql, dict(project_id=project_id))
-    return sorted(
-        row.col for row in results if is_valid_searchable_column(row.col))
+    return [row.col for row in results if is_valid_searchable_column(row.col)]
```

Comment:
How can we ensure this will give the right order that user enters?

Reply:
perhaps we should change this back?

Reply:
yes, make sense. updated